src/app/components/ElectronicDelivery/ElectronicDelivery.jsx

```diff
@@ -64,31 +64,15 @@ class ElectronicDelivery extends React.Component {
       itemId,
     } = this.state;
     const path = `/hold/confirmation/${bibId}-${itemId}`;
-
-    // console.log(fields)
-    // This will give you the form values in the form of:
-    // {
-    //   name: '',
-    //   email: '',
-    //   chapter: '',
-    //   author: '',
-    //   date: '',
-    //   volume: '',
-    //   issue: '',
-    //   'starting-page': '',
-    //   'ending-page': '',
-    // };
-    // This can then be serialized and sent to the Request API endpoint once we get it.
-    // For now it's just functionally getting this data for a client side ajax EDD request.
-    // Please delete this later.
+    const data = _extend({ bibId, itemId }, fields);
```

Comment:
This is fine. But, notice that in the `post` you are making you left `?itemId=${itemId}`. So you are adding the `itemId` as a query AND as part of the body data. If you're going to extend, remove the query and also add `pickupLocation` as part of the `data` object so it can all go together and be consistent as how we are passing data.